src/components/eventCard/EventCard.css

```diff
@@ -0,0 +1,57 @@
+.codeutsava__section3-events{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: row;
+    width: 100%;
+}
+
+.codeutsava__section3-events-container{
+    display: flex;
+    flex-direction: row;
+    width: max-content;
+    overflow-x: scroll;
+    -ms-overflow-style: none;
+    scrollbar-width: none; 
+    gap: 3rem; 
+}
+
+.codeutsava__section3-events-container::-webkit-scrollbar {
+    display: none;
+}
+
+.codeutsava__section3-card{
+    /* background-color: blueviolet; */
+    padding-bottom: 0;
+    box-sizing: border-box;
+    font-family: 'Poppins', sans-serif; 
+}
+
+.codeutsava__section3-cardtitle{
+    font-weight:bolder;
```

Comment:
Set font-weight to 600 , please look at figma design. You can click on the text in design then go to the inspect tab on top right and use the css provided for reference